templates/docker-compose.template

```diff
@@ -76,6 +78,10 @@ services:
       - SYS_RAWIO
     devices:
       - {{I2C_DEVICE}}:{{I2C_DEVICE}}
+      - {{MODEM_SERIAL0}}:{{MODEM_SERIAL0}}
```

Comment:
I'm still surprised this doesn't break on devices without modems. Have you tested that @pritamghanghas?

Reply:
the container starts. The /dev/i2c device is there and as expected /dev/ttyUSB* are missing. What other signs of trouble should I look for?

Reply:
Then this is working. I was expecting devices without modems to get stuck in a restart loop trying to detect ttyUSB, but I guess not.